pkg/ibm/ibmp.go

```diff
@@ -114,26 +119,26 @@ func (r IBMPowerDynamicConfig) authenticatedService(ctx context.Context, kubeCli
 }
 
 func (r IBMPowerDynamicConfig) GetInstanceAddress(kubeClient client.Client, ctx context.Context, instanceId cloud.InstanceIdentifier) (string, error) {
-	log := logr.FromContextOrDiscard(ctx)
+	log := logr.FromContextOrDiscard(ctx).WithValues("platform", "ibm-p", "instanceId", instanceId)
 	service, err := r.authenticatedService(ctx, kubeClient)
 	if err != nil {
 		return "", err
 	}
 	ip, err := r.lookupIp(ctx, service, string(instanceId))
 	if err != nil {
-		log.Error(err, "Failed to lookup IP", "instanceId", instanceId, "error", err.Error())
+		log.Info("Failed to lookup IP", "error", err.Error())
```

Comment:
IMO we should keep logging errors with `Error`

Reply:
ok